src/ontology/mondo-ingest.Makefile

```diff
@@ -422,19 +426,27 @@ tmp/merged.db: tmp/merged.owl
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
 
-../mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py tmp/merged.db
-	rm -f ../mappings/mondo-sources-all-lexical.sssom.tsv
-	rm -f ../mappings/mondo-sources-all-lexical-2.sssom.tsv
-	python $< run tmp/merged.db -c metadata/mondo.sssom.config.yml -r config/mondo-match-rules.yaml -o $@
+$(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py tmp/merged.db $(MAPPINGSDIR)/rejected-mappings.sssom.tsv
```

Comment:
Definitely looks a lot more succinct! Pretty big refactor in this PR.
I see a lot of usage of `ALL_COMPONENT_IDS`. My only comment is that the `mondo-sources-*-lexical*.sssom` outputs are coming out with all ontologies in 1 file without a previous step of 1 file per ontology. I don't have any qualms with that though so long as everything is working as intended!

Reply:
The split happens down the pipeline.